did-git-spec.md

```diff
@@ -148,12 +152,12 @@ TODO:
 
 **1. Repository DID**
 
-The DID for the repository can be deleted by removing the `.did/repo.did` file from the respository and performaing a commit.  The signature of this commit should correspond to one of the maintainers in the `.did/repo.did#authentication list. 
-    
+The DID for the repository can be deleted by removing the `.did/repo.did` file from the respository and performaing a commit.  The signature of this commit should correspond to one of the maintainers in the `.did/repo.did#authentication list.
```

Comment:
Again, `did/repo.did` and `did/repo.did#authenticationlist` not `.did`

Reply:
@phoniks ping?

Reply:
I rebased and pushed a commit to my branch.  I'm not seeing the changes here though.  Can you let me know if they show up for you?

Reply:
should also be fixed by rebase